src/Cardano/Wallet/Kernel/Wallet.hs

```diff
@@ -0,0 +1,119 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE ExistentialQuantification #-}
+{-# LANGUAGE TypeOperators #-}
+
+-- |
+-- Copyright: © 2018-2019 IOHK
+-- License: MIT
+--
+-- | Module provides internal implementation of
+-- | wallet api
+
+module Cardano.Wallet.Kernel.Wallet where
```

Comment:
Let's put that in `Cardano.WalletLayer` :+1:

Reply:
done, also added `Cardano.WalletLayer` to expose corresponding interface